models/notification_test.go

```diff
@@ -16,8 +16,12 @@ func TestCreateOrUpdateIssueNotifications(t *testing.T) {
 
 	assert.NoError(t, CreateOrUpdateIssueNotifications(issue, 2))
 
-	// Two watchers are inactive, thus only notification for user 10 is created
-	notf := AssertExistsAndLoadBean(t, &Notification{UserID: 10, IssueID: issue.ID}).(*Notification)
+	// 9 are inactive, thus notifications for user 1 and 4 is created
```

Comment:
nit: change to `// User 9 is inactive, ...`. The current version could mistakenly be interpreted as "9 users are inactive".

Reply:
done.